openfisca_us/parameters/gov/states/mo/tax/income/credits/mo_property_tax_credit_income_limits.yaml

```diff
@@ -0,0 +1,21 @@
+description: Income limits for MO Property Tax Credit eligibility
+#for renting joint filers the amount is technically $27_200 at first but $2000 is subtracted due to
+#their particular status. $4000 is subtracted for owning joint filers. See Line 9 of MO-PTS.
+RENTER:
+  SINGLE: 
```

Comment:
Break out each filing status (including head of household / married filing separate / widow)

Then add a breakdown metadata field

Reply:
this is based on a combination of co-habitation and filing status, i.e. one could be married filing jointly but if you did not co-habitate you would be considered single - not sure how to handle, maybe a variable?